process-engine/consumer_api_core
# ✨ Implement UserTask Functionality, LaneClaims and Correlations

_✔︎ waited on https://github.com/process-engine/consumer_api_contracts/pull/8_

## What did you change?

I added the necessary functionality to
- List userTasks
- Finish userTasks
- Handle lanes as claims
- Handle correlations

## How can others test the changes?

- checkout consumer_api_meta on `feature/user_tasks`
- run `./setup.sh`
- cd into `_integration_tests`
- have a postgres-db setup (see skeleton-database)
- run `npm start`
- wait for the bootstrapper to start
- stop the program
- run npm test

## PR-Checklist

Please check the boxes in this list after submitting your PR:
